### PR TITLE
Fixes for update 0.2.0.6

### PR DESCRIPTION
**Pull request by thijsvanloef**
## Context 
Include new PalworldSettings.ini changes as environment variables
Change pidofs to PalServer-Linux-Shipping
Update arm64 to PalServer-Linux-Shipping

## Test instructions

1. 

## Checklist before requesting a review

- [x] I have performed a self-review/test of my code
- [x] I've added documentation about this change to the [docs](https://github.com/thijsvanloef/palworld-server-docker/tree/main/docusaurus/docs).
- [ ] I've not introduced breaking changes.
- [x] My changes do not violate linting rules.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -219,9 +219,13 @@ It is highly recommended you set the following environment values before startin
...
 | RCON_PORT                                  | RCON port to connect to                                                                                                                                                                             | 25575                                                                                              | 1024-65535                                                                                                        |
+| REST_API_ENABLED                                  | Enable REST API for the palworld server                                                                                                                                                                             | false                                                                                              | 1024-65535                                                                                                        |
```
Comment (Dashboy1998) [inline README.md:222]:
```suggestion
| REST_API_ENABLED                                  | Enable REST API for the palworld server                                                                                                                                                                             | false                                                                                              | true/false                                                                                                        |
```

File: docusaurus/docs/getting-started/configuration/server-settings.md [hunk 2]
```diff
@@ -39,7 +39,10 @@ It is highly recommended you set the following environment values before startin
...
-| QUERY_PORT                                 | Query port used to communicate with Steam servers                                                                                                                                                   | 27015                                                                                                  | 1024-65535                                                                                                                                      |
+| REST_API_ENABLED                                  | Enable REST API for the palworld server                                                                                                                                                                             | false                                                                                              | 1024-65535                                                                                                        |
```
Comment (Dashboy1998) [inline docusaurus/docs/getting-started/configuration/server-settings.md:42]:
```suggestion
| REST_API_ENABLED                                  | Enable REST API for the palworld server                                                                                                                                                                             | false                                                                                              | true/false                                                                                                        |
```

File: scripts/compile-settings.sh [hunk 3]
```diff
@@ -150,7 +154,11 @@ RCON_PORT = $RCON_PORT
 REGION = $REGION
 USEAUTH = $USEAUTH
 BAN_LIST_URL = $BAN_LIST_URL
+REST_API_ENABLED= $REST_API_ENABLED
+REST_API_PORT= $REST_API_PORT
 SHOW_PLAYER_LIST = $SHOW_PLAYER_LIST
+ALLOW_CONNECT_PLATFORM= $ALLOW_CONNECT_PLATFORM
+USE_BACKUP_SAVE_DATA= $USE_BACKUP_SAVE_DATA
```
Comment (Dashboy1998) [inline scripts/compile-settings.sh:161]:
```suggestion
REST_API_ENABLED = $REST_API_ENABLED
REST_API_PORT = $REST_API_PORT
SHOW_PLAYER_LIST = $SHOW_PLAYER_LIST
ALLOW_CONNECT_PLATFORM = $ALLOW_CONNECT_PLATFORM
USE_BACKUP_SAVE_DATA = $USE_BACKUP_SAVE_DATA
```

Keep it consistent

**Review by Dashboy1998 — CHANGES_REQUESTED** [review]

File: scripts/start.sh [hunk 4]
```diff
@@ -64,7 +64,7 @@ if [ "$architecture" == "arm64" ]; then
             ;;
     esac
     
-    sed -i "s|\(\"\$UE_PROJECT_ROOT\/Pal\/Binaries\/Linux\/PalServer-Linux-Test\" Pal \"\$@\"\)|LD_LIBRARY_PATH=/home/steam/steamcmd/linux64:\$LD_LIBRARY_PATH $box64_binary \1|" ./PalServer-arm64.sh
+    sed -i "s|\(\"\$UE_PROJECT_ROOT\/Pal\/Binaries\/Linux\/PalServer-Linux-Shipping\" Pal \"\$@\"\)|LD_LIBRARY_PATH=/home/steam/steamcmd/linux64:\$LD_LIBRARY_PATH $box64_binary \1|" ./PalServer-arm64.sh
```
Comment (Dashboy1998) [inline scripts/start.sh:67]: Do we want to have an if statement to support older versions?

Comment (thijsvanloef) [reply scripts/start.sh:67]: No I don't think so, people wanting to stay on a specific version of the game, would have to pin the container version. 
Backwards compatibility is very hard to support in an early access game .

**Review by Dashboy1998 — APPROVED** [review]

**Pull request merged**